app/helpers/sessions_helper.rb

```diff
@@ -15,4 +23,22 @@ def log_out
     session.delete :user_id
     @current_user = nil
   end
+
+  def remember user
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
```

Comment:
I have a question: What is the purpose of using signed cookie here? Why it is useful ?

Reply:
use signed cookie to store session data(user id) can avoid an attack session hacking